src/test/java/api/support/APITests.java

```diff
@@ -228,6 +235,7 @@ public void beforeEach()
     holdingsClient.deleteAll();
     instancesClient.deleteAll();
     configClient.deleteAll();
+    expiredEndSessionClient.deleteAll();
```

Comment:
Does this really need to be done for every API test, and not only the ones specific to sessions?

Reply:
`expiredEndSessionClient.deleteAll();` has been moved to EndExpiredPatronActionSessionTests